docs/docs/en/architecture/design.md

```diff
@@ -55,7 +55,7 @@
 
   - **WorkerManagerThread** is mainly responsible for the submission of the task queue, continuously receives tasks from the task queue, and submits them to the thread pool for processing;
 
-  - **TaskExecuteThread** is mainly responsible for the process of task execution, and the actual processing of tasks according to different task types;
+  - **TaskExecuteThread** is mainly responsible for the workflow of task execution, and the actual processing of tasks according to different task types;
```

Comment:
personally think this should not change too

Reply:
@zhongjiajie Please confirm whether it needs to be modified, thank you

Reply:
No, shouldn't change it. Keep it as `process`.

Reply:
> No, shouldn't change it. Keep it as `process`.
OK, I understand